docs/architecture/decisions/0003-es-aaq-documents.md

```diff
@@ -0,0 +1,72 @@
+# 3. AAQ structure in Search v2
+
+Date: 2020-10-27
+
+## Status
+
+Accepted
+
+## Context
```

Comment:
I would also add something about the architecture itself which is different from what exists now (denormalization)

Reply:
Do we use nested/joins currently? I'm not sure we do, we just needed to be cautious not to introduce it while re-modelling our data. I'll add a note about that.